js/pages/incidence-rates/tabs/ir-export.html

```diff
@@ -0,0 +1,57 @@
+<div data-bind="css: classes()">
+  <div data-bind="css: classes('export-btn')">
+    <span id="copyCopyIRSQLMessage" style="display:none" class="alert alert-success" role="alert"><i class="fa fa-check-square-o" aria-hidden="true"></i><strong>Copied To Clipboard!</strong>
+    </span>&nbsp;&nbsp;<button class="btn btn-sm btn-primary" itle="Copy to clipboard" id="btnCopyIRSQLClipboard" data-bind="click: parent.copyIRSQLToClipboard" data-clipboard-target="#irSQL">
+      <i class="fa fa-clipboard" aria-hidden="true"></i> Copy To Clipboard</button>
+  </div>
+  <div id='irSQL'>
+    <tabs params="
+        tabs: [
```

Comment:
Why not to point tabs in the same way as in the cohort definition page?

Reply:
tabs/ir-export.* were removed from sources